src/pages/storage/StorageBuckets.tsx

```diff
@@ -255,6 +256,10 @@ const StorageBuckets: FC = () => {
     return selectedNames.includes(bucketKey);
   });
 
+  const panelBucket = buckets.find((bucket) => {
+    return bucket.name == panelParams.bucket;
```

Comment:
To identify a bucket we need the name, pool and location. So this identifier here needs to check on all three.